Changes

```diff
@@ -1,5 +1,6 @@
 {{$NEXT}}
 
+- #15: Fixed issue where save wouldn't work if called during global destruction
```

Comment:
I like this, and it all looks pretty good. However, I'd like to see a unit test that shows all transactions having been recorded without the thing going out of scope.